apps/cloudinary/src/index.js

```diff
@@ -76,8 +76,13 @@ function renderDialog(sdk) {
   const instance = cloudinary.createMediaLibrary(options, {
     insertHandler: data => sdk.close(data)
   });
+  const show_options={};
 
-  instance.show();
+  if (typeof(config.startFolder) !== 'undefined' && config.startFolder != "") {
```

Comment:
I believe we should harden this check to ensure that a string is being passed.
```suggestion
  if (typeof config.startFolder === 'string' && config.startFolder.length) {
```

This will also rule out any null/undefined values as well.